coalib/bears/PackageRequirement.py

```diff
@@ -19,3 +19,15 @@ def check(self):
         Check if the requirement is satisfied.
         """
         raise NotImplementedError
+
+
+class PythonRequirement(PackageRequirement):
+
+    def __init__(self, package, version):
+        PackageRequirement.__init__(self, 'pip', package, version)
+
+    def __str__(self):
+        return "{.package} version {.version}".format(self)
+
+    def multiple(self, *args):
+        return args
```

Comment:
Shouldn't this be a classmethod that returns a tuple of `PythonRequirement` instances?